CDP4Orm.Tests/EncryptionUtilsTestFixture.cs

```diff
@@ -56,5 +57,22 @@ public void VerifyThatByteConversionWorks()
 
             Assert.AreEqual(password, returned);
         }
+
+        [Test]
+        public void VerifyThatWspPasswordCompareWorks()
```

Comment:
should this not be in another test project, the WSP authentication test project?

Reply:
New test project was added in order to accomplish this.